### PR TITLE
Review how to join

### PR DESCRIPTION
**Pull request by DavidCroftDKFZ**

### CODE REVIEW

File: fsh/ig-data/input/pages/workplace.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```
Comment (alexanderkiel) [inline fsh/ig-data/input/pages/workplace.html]: Did you create this file internationally?

**Review by alexanderkiel — CHANGES_REQUESTED** [review]

File: fsh/ig-data/input/pages/workplace.html (hunk 1 above)
Comment (DavidCroftDKFZ) [reply fsh/ig-data/input/pages/workplace.html]: I'm not sure what you mean here, could you expand on your comment?

Comment (alexanderkiel) [reply fsh/ig-data/input/pages/workplace.html]: You added this file, but why?

Comment (DavidCroftDKFZ) [reply fsh/ig-data/input/pages/workplace.html]: Oh - I see! I thought I had removed it already, I was using it during the testing phase.

I will delete it.

Comment (DavidCroftDKFZ) [reply fsh/ig-data/input/pages/workplace.html]: ...it's now gone!

**Review by alexanderkiel — APPROVED** [review]

**Pull request merged**